Source/WebKit/WebProcess/Network/WebTransportSession.cpp

```diff
@@ -165,4 +170,22 @@ void WebTransportSession::networkProcessCrashed()
     if (RefPtr strongClient = m_client.get())
         strongClient->networkProcessCrashed();
 }
+
+void WebTransportSession::cancelReceiveStream(WebCore::WebTransportStreamIdentifier identifier, std::optional<WebCore::WebTransportStreamErrorCode> errorCode)
+{
+    ASSERT(RunLoop::isMain());
```

Comment:
I don't think this is correct.  WorkerWebTransportSession::cancelReceiveStream calls this directly from the non-main run loop.  I'd just remove these.